apps/system/js/task_manager.js

```diff
@@ -538,9 +553,38 @@ TaskManager.prototype = {
       this.publish('taskmanager-deactivated');
     }
     this.element.classList.toggle('active', active);
-    this.element.classList.toggle('empty', active && this.stack.length === 0);
   },
 
+  /**
+   * Launch a new browser sheet at the end of the stack, pan to its position,
+   * and exit into the new sheet.
+   */
+  openNewSheet({ isPrivate } = {}) {
+    var appWindow;
+
+    if (isPrivate) {
+      appWindow = new AppWindow(new BrowserConfigHelper({
+        manifestURL: 'app://search.gaiamobile.org/manifest.webapp',
+        url: 'app://search.gaiamobile.org/newtab.html?private=1',
+        isPrivate: true
+      }));
```

Comment:
looks like this is not actually opening a private window :)
(the private window have a different visual treatment for chrome and in the card view, you can use the "..." menu in the browser chrome to open one and see the different).

Haven't dug into the details but https://github.com/mozilla-b2g/gaia/blob/49a75c53b3119cf73f1dcec2926fe4a8ebde9da9/apps/system/js/browser.js#L59-L70 works